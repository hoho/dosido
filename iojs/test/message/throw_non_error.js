'use strict';
var common = require('../common');
var assert = require('assert');

console.error('before');

// custom error throwing
throw ({ foo: 'bar' });

console.error('after');
