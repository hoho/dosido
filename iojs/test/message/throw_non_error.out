before
*test*message*throw_non_error.js:8
throw ({ foo: 'bar' });
^
[object Object]
